data-prepper-plugins/otel-trace-source/src/main/java/com/amazon/dataprepper/plugins/certificate/acm/ACMCertificateProvider.java

```diff
@@ -76,14 +86,28 @@ public Certificate getCertificate() {
             }
             timeSlept += SLEEP_INTERVAL;
         }
-        if(exportCertificateResult != null) {
+        if (exportCertificateResult != null) {
             final String decryptedPrivateKey = getDecryptedPrivateKey(exportCertificateResult.getPrivateKey(), pkPassphrase);
             return new Certificate(exportCertificateResult.getCertificate(), decryptedPrivateKey);
         } else {
-            throw new IllegalStateException(String.format("Exception retrieving certificate results. Time spent retrieving certificate is %d ms and total time out set is %d ms.", timeSlept, totalTimeout));
+            throw new IllegalStateException(String.format("Exception retrieving certificate results. Time spent retrieving certificate is" +
+                    " %d ms and total time out set is %d ms.", timeSlept, totalTimeout));
         }
     }
 
+    private String generatePassphrase(final int characterCount) {
```

Comment:
QUES: what is the advantage of generate pass phrase this way compared to random UUID?